### ISSUE TITLE
Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile) (Tues 6 May)

### ISSUE THREAD

**Issue by tamsinewing555** [w3c/wai-reviews-and-announcements#6]
<h2>Document details</h2>

- **Last published document: **
  - None for this doc
  - previous doc [Mobile Accessibility: How WCAG 2.0 and Other W3C/WAI Guidelines Apply to Mobile](https://www.w3.org/TR/mobile-accessibility-mapping/) 
-  **Staged document:** https://w3c.github.io/matf/ (reviewed Editor's Draft was published on 2 April)
- **Version/Type of document:** Group Draft Note
- **W3C Staff Contact:** Kevin White
- **Task Force:** [Mobile Accessibility Task Force](https://www.w3.org/groups/tf/mobile-a11y-tf/)
- **Working Group:**  [Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/)
- **Editor(s):** Jan Jaap de Groot (Abra), Jamie Herrera (Invited Expert), Joe Humbert (Invited Expert), Julian Kittelson-Aldred (Invited Expert), Jon Gibbins (Invited Expert)

<h2>Review</h2>

* This document was reviewed: https://docs.google.com/document/d/17lBz_ejeGZHemRHVzRZKQmygKbntREX5EjLoin8Fy6U/edit?tab=t.0#heading=h.alchsoozkv9j

<h2>Announcements</h2>

- **Announcements needed?:** Yes
- **Announcement channels:**
  - WAI email
  - WAI news
  - W3C news
  - WAI LinkedIn - No?
  - WAI Mastodon - No?
- **Draft announcement blurbs provided in:** [Google Doc](https://docs.google.com/document/d/1VzgBvMPho3RStEuIrsCPkNrxQHIzDQxnQD83vw1lf1k/edit?usp=drive_link)
- **Signatories:** 
- **Target date for announcement:** 2025-04-24 (if possible)
- **Date public feedback is due by:** 

<h3>Key messages:</h3>

* Work in progress
* Seeking input on the direction and approach
* WCAG applies to mobile, WAI does not have separate mobile standards
* WCAG2ICT provides high-level guidance on applying WCAG 2 to ICT including mobile apps; WCAG2Mobile follows on from WCAG2ICT and provides more specific guidance
* Point people to this as the starting place for W3C mobile accessibility info: [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/)

<h3>Summary of changes since last publication:</h3>

n/a (first publication)

<h3>Review questions</h3>

* Does the structure of the guidance, mostly related to the Success Criteria, e.g. expand/collapse for WCAG + WCAG2ICT, followed by MATF guidance work?
* Is the wording in our guidance easy-to-read and unambiguous?
* Is there anything missing in the approach being taken?

<h2>Related docs that need updating:</h2>

* [Mobile Accessibility at W3C](https://www.w3.org/WAI/standards-guidelines/mobile/)
  * SLH: JJ and Kevin are suppose to take a pass at this, though if it's just simple updates, then Tamsin could take a pass 
(TE: Done: https://docs.google.com/document/d/1cGPIZ8nUVQNKzT5V3oMrFWM7NpO-m-9JCP3LAPX46Y4/edit?tab=t.0#heading=h.mexcbdyzqw14)
* Mobile Accessibility Task Force page - Publications section
  * SLH: A major goal for the WG/TF project was *not* having to update these pages on the WAI website; I hope it's not there and instead on a wiki or other page the JJ can update himself
TE: We won't have to update versions. We just need to add the new documents (each guidance document would eventually have the technical report and the Editor's Draft). Having these on the WG pages was identified as a user need in the user interviews. Adding new documents is now part of the review and publish process, along with the minor updates to other related documents.

<h2>Publication links</h2>

- Final published document: https://www.w3.org/TR/wcag2mobile-22/
- Links to email and social media announcements: [link after review]
  - WAI email public-announce: https://lists.w3.org/Archives/Public/public-wai-announce/2025AprJun/0001.html 
  - WAI news: https://www.w3.org/WAI/news/2025-05-06/wcag2mobile/
  - W3C news: https://www.w3.org/news/2025/draft-note-for-review-guidance-on-applying-wcag-2-2-to-mobile-applications-wcag2mobile/ 

<h2>Managing the review and publishing process</h2>

 - Date the announcement wording needs to be ready for review by Team Contact:
 - Date the PRs for the WAI news blurb and the web page updates need to be committed for approval:

<h2>Notes:</h2>

* [x] doc title — Shawn's input was considered and the issue is closed

**Comment by iadawn** [comment]
Version/Type of document: Draft note
W3C Staff Contact: Kevin

I have reviewed and commented on editorial pass for abstract and introduction. Comments added.

Is there an announcement to review?

PR created to add editorial updates to: w3c/matf#100

**Comment by iadawn** [comment]
> Version/Type of document: first working draft

This is a 'Draft Note'.

**Comment by iadawn** [comment]
Would be useful to have something like 'estimated date of publication' or 'desired date of publication'. I think having it in the title is distracting?

**Comment by tamsinewing555** [comment]
> Would be useful to have something like 'estimated date of publication' or 'desired date of publication'. I think having it in the title is distracting?

The 'Target date for announcement' is under the 'Announcement' heading.

Shawn suggested I also put the date in the title to help us all quickly identify which ones have priority.
But happy to remove it if it's distracting.
How does it look to you now I've moved the date to the end of the title?
